functions.php

```diff
@@ -881,4 +881,34 @@ public function template_json( $template ) {
 
 $ucf_feed_json = new UCF_Feed_JSON();
 
+function post_get_thumbnail( $object, $field_name, $request ) {
+	$image = null;
+
+	if ( $object['featured_media'] ) {
+		$image = wp_get_attachment_image_src( $object['featured_media'] );
+	} else {
+		$attachments = get_attached_media( 'image', $object['id'] );
+		if ( $attachments ) {
+			foreach( $attachments as $key=>$val ) {
+				if ( $image = wp_get_attachment_image_src( $key ) ) {
+					break;
+				}
+			}
+		}
+	}
+
+	return $image[0];
```

Comment:
$image[0] will return an error if $image is false--check that $image is an array before accessing the first index here